components/html-block/demo/html-block.html

```diff
@@ -261,6 +261,168 @@ <h2>HTML Block (inline math)</h2>
 				</template>
 			</d2l-demo-snippet>
 
+			<h2>HTML Block (wrapped)</h2>
+
+			<d2l-demo-snippet>
+				<template>
+					<d2l-some-component></d2l-some-component>
+				</template>
+			</d2l-demo-snippet>
+
+			<d2l-code-view language="javascript">
```

Comment:
I don't love that this is basically a copy of the actual demo code, but this is due to some limitations of the demo-snippet.  On the plus side, it does allow me to construct the snippet in a summarized fashion, excluding the less interesting bit of the code.